comps/guardrails/pii_detection/README.md

```diff
@@ -0,0 +1,78 @@
+# PII Detection Microservice
+
+PII Detection a method to detect Personal Identifiable Information in text. This microservice provides users a unified API to either upload your files or send a list of text, and return with a list following original sequence of labels marking if it contains PII or not.
+
+# 🚀1. Start Microservice with Python（Option 1）
+
+## 1.1 Install Requirements
+
+```bash
+pip install -r requirements.txt
+```
+
+## 1.2 Start PII Detection Microservice with Python Script
+
+Start pii detection microservice with below command.
+
+```bash
+python pii_detection.py
+```
+
+# 🚀2. Start Microservice with Docker (Option 2)
+
+## 2.1 Prepare PII detection model
+
+export HUGGINGFACEHUB_API_TOKEN=${HP_TOKEN}
+
+## 2.1.1 use LLM endpoint (will add later)
+
+intro placeholder
+
```

Comment:
Hi, @ftian1 , I removed the model downloading from README, so it will not have conflict for ospdt perspective.